week_1/String/longest_common_prefix.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param {string[]} strs
+ * @return {string}
+ */
+const longestCommonPrefix = (words) => {
+	let longest_common_prefix = "";
+	outer: for (letter = 0; letter < words[0].length; letter++) {
+		for (word = 1; word < words.length; word++) {
```

Comment:
Without an issue or broader description, I don't know what the goals of this function are. Can we link that in the PR?